haystack/nodes/file_converter/json.py

```diff
@@ -0,0 +1,95 @@
+import json
+from json import JSONDecodeError
+
+from typing import List, Optional, Any, Dict
+
+import logging
+from pathlib import Path
+
+from haystack import Document
+from haystack.nodes.file_converter import BaseConverter
+
+
+logger = logging.getLogger(__name__)
+
+
+class JsonConverter(BaseConverter):
+    """Extracts text from JSON files and casts it into Document objects."""
+
+    outgoing_edges = 1
+
+    def convert(
+        self,
+        file_path: Path,
+        meta: Optional[Dict[str, Any]] = None,
+        remove_numeric_tables: Optional[bool] = None,
+        valid_languages: Optional[List[str]] = None,
+        encoding: Optional[str] = "UTF-8",
+        id_hash_keys: Optional[List[str]] = None,
+    ) -> List[Document]:
+        """
+        Reads a JSON file and converts it into a list of Documents.
+
+        It's primarily a wrapper around `Document.from_dict()` and, as such, acts as the inverse of `Document.to_dict()`.
```

Comment:
When you say `primarily` it sounds like it mostly behaves as `Document.from_dict()`, but not exactly. Would it be fairer to just say, `It's a wrapper around ...`? Otherwise as a user I might think it has extra features or differences compared to `from_dict`. And if it does it would be good to explain what those differences are here.

Reply:
Good point!